app/src/main/java/org/mozilla/fenix/tabstray/browser/RemoveTabUseCaseWrapper.kt

```diff
@@ -7,29 +7,17 @@ package org.mozilla.fenix.tabstray.browser
 import mozilla.components.feature.tabs.TabsUseCases
 import org.mozilla.fenix.GleanMetrics.TabsTray
 
-class SelectTabUseCaseWrapper(
-    private val selectTab: TabsUseCases.SelectTabUseCase,
-    private val onSelect: (String) -> Unit,
-) : TabsUseCases.SelectTabUseCase {
-    operator fun invoke(tabId: String, source: String? = null) {
-        TabsTray.openedExistingTab.record(TabsTray.OpenedExistingTabExtra(source ?: "unknown"))
-        selectTab(tabId)
-        onSelect(tabId)
-    }
-
-    override fun invoke(tabId: String) {
-        invoke(tabId, null)
-    }
-}
-
+@Suppress("UndocumentedPublicClass")
 class RemoveTabUseCaseWrapper(
     private val onRemove: (String) -> Unit,
 ) : TabsUseCases.RemoveTabUseCase {
+    @Suppress("UndocumentedPublicFunction")
     operator fun invoke(tabId: String, source: String? = null) {
         TabsTray.closedExistingTab.record(TabsTray.ClosedExistingTabExtra(source ?: "unknown"))
         onRemove(tabId)
     }
 
+    @Suppress("UndocumentedPublicFunction")
```

Comment:
I was getting lint errors with these after removing the select tab wrapper, so had to add some temporary `@Supress`. 

This will be removed subsequently [here](https://bugzilla.mozilla.org/show_bug.cgi?id=1814223).

Reply:
This might be because you need to update the detekt-baseline? https://searchfox.org/mozilla-mobile/source/fenix/detekt-baseline.xml#505

Reply:
which would be more preferable? `RemoveTabUseCaseWrapper` isn't actually being used anywhere, as you pointed out below, so it's a temporary band-aid either way until this is cleaned up this week or next.

Reply:
I think it's fine since we're removing it anyways. Just wanted to point out the baseline since you might want to also update the baseline when you also remove it.